### DISCUSSION TITLE
Update from lazysize.js to lazysizes.min.js

### DISCUSSION THREAD

**Discussion by unic0rned** [aFarkas/lazysizes#875] [Q&A]
Hey,

my shopify store has an old 4.1 something version of lazysize.js installed.
I want to update to lazysizes.min.js but if i replace the old script in the theme.liquid and upload the new asset, my pictures on the website get broken.

Also somehow lazyloading right now is a litte bit weird.

Does someone know how i can bulk add lazyload class to all my IMG tags in Shopify?
I really don't know how to.

Thanks

**Comment by 4li1985** [comment]
Does this help?
https://easycodeguide.com/add-lazy-load-image-shopify-theme.html

**Reply by unic0rned** [reply]
A little bit.

But the problem remains.
My pictures have a lazyloaded class and not the lazyload class.

How can I put the lazyload class on all of them in Shopify?

**Reply by unic0rned** [reply]
The problem is we use sections pro.
So our website has tons of sections. I can't manually edit all IMG Tags.

Can I do it via liquid or javascript?
